nearswap/src/deposit.rs

```diff
@@ -49,11 +47,14 @@ impl NearSwap {
     Deposits attached NEAR.
     Panics if the sender account is not registered. */
     #[payable]
-    pub fn deposit_near(&mut self) {
+    pub fn deposit_near(&mut self, registration_only: Option<bool>) {
```

Comment:
As noted in the comment below, let's don't use `registration_only` here.

Reply:
updated